gnomad_qc/v3/create_release/make_var_annot_hists.py

```diff
@@ -0,0 +1,65 @@
+from gnomad_hail import try_slack
+from gnomad_hail.utils.annotations import get_annotations_hists, create_frequency_bins_expr
+from gnomad_hail.utils.constants import ANNOTATIONS_HISTS
+import hail as hl
+import argparse
+from gnomad_qc.v3.resources import qual_hists_json_path, release_ht_path
+
+
+def main(args):
+    hl.init(default_reference='GRCh38', log='/variant_histograms.log')
+
+    ht = hl.read_table(release_ht_path())
+    # NOTE: histogram aggregations are done on the entire callset (not just PASS variants), on raw data
+
+    hist_dict = ANNOTATIONS_HISTS
+    hist_dict['MQ'] = (20, 60, 40) # Boundaries changed for v3, but could be a good idea to settle on a standard
+    hist_ranges_expr = get_annotations_hists(
+        ht,
+        ANNOTATIONS_HISTS
+    )
+
+    # NOTE: run the following code in a first pass to determine bounds for metrics
```

Comment:
it might be helpful to mention this part of the code doesn't actually need to be run each time